### DISCUSSION TITLE
Removing installed themes

### DISCUSSION THREAD

**Discussion by ollilukkari** [t1m0thyj/WinDynamicDesktop#412] [Q&A]
Hey, 
for some reason the free downloadable themes "Mountains #2", "Mountains #3" don't work for me, 
WinDD throws an error notification. I tried overwriting/reimporting them but the issue persists.
How do I remove these specific wallpapers to not get the error in the future? Below is a snippet from the error log file, unfortunately some of it is Finnish as my Windows is but maybe it helps. Win11 by the way.

[2022-02-09 14:30:40.413] System.IO.FileNotFoundException: Määritettyä tiedostoa ei löydy. (HRESULT-poikkeus: 0x80070002)
   kohteessa System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   kohteessa System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   kohteessa WinDynamicDesktop.UwpHelper.<SetWallpaper>d__5.MoveNext()
--- Pinon jäljityksen loppu aiemmasta sijainnista, jossa palautettiin poikkeus ---
   kohteessa System.Runtime.CompilerServices.AsyncMethodBuilderCore.<>c.<ThrowAsync>b__6_0(Object state)

**Comment by t1m0thyj** [comment]
This may be related to https://github.com/t1m0thyj/WinDynamicDesktop/issues/434
